src/project/qml/MuseScore/Project/ScoresPage.qml

```diff
@@ -139,6 +139,8 @@ FocusScope {
 
         backgroundColor: background.color
 
+        isSearching: searchField.focus && searchField.searchText.length != 0
```

Comment:
Personally I'd drop `searchField.focus &&`, because otherwise the "no results" item would disappear as soon as the user clicks outside the search field, and I think that might be confusing.

Reply:
I updated the PR :)